Crashing when browsing channel

### Steps to reproduce

Hi guys,

The latest version is crashing when browsing any yt channels videos.

Here is the log:

kotlinx.serialization.MissingFieldException: Fields [id, name, avatarUrl, bannerUrl, description] are required for type with serial name 'com.github.libretube.api.obj.Channel', but they were missing at path: $
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue$1(StreamingJsonDecoder.kt:107)
	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:25)
	at com.jakewharton.retrofit2.converter.kotlinx.serialization.Serializer$FromString.fromResponseBody(Serializer.kt:52)
	at com.jakewharton.retrofit2.converter.kotlinx.serialization.DeserializationStrategyConverter.convert(DeserializationStrategyConverter.kt:12)
	at retrofit2.OkHttpCall.parseResponse(OkHttpCall.java:57)
	at retrofit2.OkHttpCall$1.onResponse(OkHttpCall.java:5)
	at com.google.net.cronet.okhttptransport.CronetCallFactory$CronetCall$2.onSuccess(CronetCallFactory.java:30)
	at com.google.common.util.concurrent.Futures$CallbackListener.run(Futures.java:26)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@777e2c, Dispatchers.Main.immediate]
Caused by: kotlinx.serialization.MissingFieldException: Fields [id, name, avatarUrl, bannerUrl, description] are required for type with serial name 'com.github.libretube.api.obj.Channel', but they were missing
	at com.google.common.math.MathPreconditions.throwMissingFieldException(MathPreconditions.java:115)
	at com.github.libretube.api.obj.Channel.<init>(Channel.kt:74)
	at com.github.libretube.api.obj.Channel$$serializer.deserialize(Channel.kt:176)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue$1(StreamingJsonDecoder.kt:67)
	... 10 more
    

### Expected behavior

Should not crash

### Actual behavior

Crashing

### LibreTube version

0.11.0

### Android version

13

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

--- a/app/src/main/java/com/github/libretube/api/obj/Channel.kt
+++ b/app/src/main/java/com/github/libretube/api/obj/Channel.kt
@@ -4,11 +4,11 @@ import kotlinx.serialization.Serializable
 
 @Serializable
 data class Channel(
-    val id: String,
-    val name: String,
-    val avatarUrl: String,
+    val id: String? = null,
+    val name: String? = null,
+    val avatarUrl: String? = null,
     val bannerUrl: String? = null,
-    val description: String,
+    val description: String = "",
     val nextpage: String? = null,
     val subscriberCount: Long = 0,
     val verified: Boolean = false,

--- a/app/src/main/java/com/github/libretube/api/obj/Channel.kt
+++ b/app/src/main/java/com/github/libretube/api/obj/Channel.kt
@@ -7,7 +7,7 @@ data class Channel(
     val id: String,
     val name: String,
     val avatarUrl: String,
-    val bannerUrl: String,
+    val bannerUrl: String? = null,
     val description: String,
     val nextpage: String? = null,
     val subscriberCount: Long = 0,

--- a/app/src/main/java/com/github/libretube/ui/fragments/ChannelFragment.kt
+++ b/app/src/main/java/com/github/libretube/ui/fragments/ChannelFragment.kt
@@ -142,7 +142,7 @@ class ChannelFragment : BaseFragment() {
 
                 binding.channelShare.setOnClickListener {
                     val shareDialog = ShareDialog(
-                        response.id.toID(),
+                        response.id!!.toID(),
                         ShareObjectType.CHANNEL,
                         shareData
                     )

